### PR TITLE
Shorten sign out sync conflict messaging

### LINKED ISSUE #152: Shorten sign out sync alert titles to avoid text truncation on Android
iOS doesn't truncate alert titles, so this is Android-only.

screen shot 2017-03-23 at 1 11 31 am

### PR DESCRIPTION
**Pull request by mattyu007**
Closes #152. The title is now short enough to not be truncated on Android.

Buttons are now in sync across iOS and Android, and a Cancel option to dismiss and cancel signout has been added:
screen shot 2017-03-26 at 3 31 41 pm

### CODE REVIEW

**Comment by henryc132** [comment]
It feels very unnatural to have the sign out button at the bottom

File: Cue/app/tabs/account/AccountHome.js [hunk 1]
```diff
@@ -87,7 +88,7 @@ class AccountHome extends React.Component {
       }).catch(e => {
         console.warn('Failed to sync changes', e)
         Alert.alert(
-          'Some changes haven’t been synced to the Cue cloud yet',
+          'Some Changes Haven’t Been Synced to the Cue Cloud Yet',
```
Comment (henryc132) [inline Cue/app/tabs/account/AccountHome.js:92]: Are we just going to title case everything now? Or keep the android/ios versions

Comment (Tyler-Brenneman) [reply Cue/app/tabs/account/AccountHome.js:92]: AccountHome is ios only, so only need title case here

**Comment by mattyu007** [comment]
Updated PR to normalize button order and allow dismissing the dialog without signing out or entering the Sync Conflict Resolution flow.

**Review by ClaytonPassmore — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**